src/Graph/GraphNode.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace SRF\Graph;
+
+class GraphNode {
+	private $id;
+	private $parent = [];
+	private $label;
+
+	/**
+	 * @var string $id : Node ID including namespace
+	 */
+	public function __construct( $id ) {
```

Comment:
Meh. What about bumping the min PHP version of SRF to 7.1? Travis already at 7.0 so not much of a change. @mwjames @kghbln @RacoonDev

Reply:
I think i can resolve this one...

> @RacoonDev Sorry about the release boundary discussion and the noise which is surely the wrong place here therefore I'll hide those discussion points.